### PR TITLE
Copy event

### LINKED ISSUE #21: C for Copy
Much like the current `move_image` functionality, implement a `copy_image` function that instead of moving the file, copies it to the destination folder.

* Add `Copy` to the `Action` enum in `ui.rs`
* Match `c` key to `Copy` action in `event_action` function in `ui.rs`
* Match `Action::Copy` to `copy_image` function in `run` in `program.rs`
* Update README with new controls

### PR DESCRIPTION
**Pull request by gurgalex**
Hiya!

Neat project. Great for looking through full screen images.

I've added the Copy functionality (mostly copied the move_image code).

I've added a reminder to decide what to do when the image already exists (may not want to override the image). Prompt if no option specified or use specified option may work.

Fixes #21

### CODE REVIEW

**Comment by gurgalex** [comment]
Hmm, should probably move getting the current filepath and making the destination path to a function.
Noticed that when looking at how to delete a file next.

**Comment by gurgalex** [comment]
Update: Fixed
Do not merge.
There seems to be a bug where copying gets called without a keypress when 0 images are present.

File: src/program.rs
Comment (Davejkane) [inline src/program.rs]: Should this `<` be the other way around? `>`

File: src/program.rs
Comment (Davejkane) [inline src/program.rs]: Can you match on this in the same way as `move_image`? Otherwise some error in copying the image will crash the program.

File: src/program.rs
Comment (Davejkane) [inline src/program.rs]: I don't think we need to re-render here. Maybe just return `Ok(())` instead

**Review by Davejkane — CHANGES_REQUESTED** [review]
Once again, thanks for getting involved and improving my code a lot. I really appreciate it. I've added a couple of comments, and again it's pretty minor stuff.

You're right about maybe having a prompt for what to do in the case of an image already existing. If I remember correctly, in windows doing that used to just add `(1)` to the end of the filename. e.g. `image.jpg` -> `image(1).jpg`. We could go for a solution like that? but it would mean handling `image(1).jpg` -> `image(2).jpg` etc. 

Otherwise once we have this https://github.com/Davejkane/riv/issues/5 implemented we can look at doing prompts.

File: src/program.rs
Comment (gurgalex) [reply src/program.rs]: Ah, right.
Thanks.

File: src/program.rs
Comment (gurgalex) [reply src/program.rs]: Yep!

File: src/program.rs
Comment (gurgalex) [reply src/program.rs]: Okay

Comment (Davejkane) [reply src/program.rs]: This still needs to be fixed

**Review by Davejkane — CHANGES_REQUESTED** [review]
After merging the Delete key PR to development there are now conflicts that will have to be resolved. Please fix that up and take care of not re-rendering at the end of the copy function.

**Review by Davejkane — APPROVED** [review]
Awesome. Looks great.

**Pull request merged**
